src/view/05_forms.md

```diff
@@ -144,36 +146,36 @@ Two form elements tend to cause some confusion, in different ways.
 ### `<textarea>`
 
 Unlike `<input>`, the `<textarea>` element does not support a `value` attribute.
-Instead, it receives its value as a plain text node in its HTML children,
+Instead, it receives its value as a plain text node in its HTML children.
 
-In the current version of Leptos (in fact in Leptos 0.1-0.5), creating a dynamic child
+In the current version of Leptos (in fact in Leptos 0.1-0.6), creating a dynamic child
```

Comment:
Updated this version, I think it probably hasn't changed in 0.6 but I haven't checked.